### DISCUSSION TITLE
Add "Account Connections" to your profile. Similar to Discord:-)

### DISCUSSION THREAD

**Discussion by MrLuxuri** [stoatchat/discussions#272] [Feature Suggestions]
**Account Connections** allows you to connect your reddit, twitter, GitHub, or even link your Xbox account on Discord. We could take that a step further and allow even more connections, like allowing Playstation and Nintendo Switch connections apart from Xbox which is only supported on Discord.

Why implement this?
Example; Xbox connections changes your status automatically when you are playing a game on your xbox at Discord. This would be awesome across all console platforms for Revolt.

**Comment by exdnaa** [comment]
I think a Spotify Integration would be great.

**Reply by enygmate** [reply]
I'd like to add to the Spotify Integration, the listening parties and rich presences just like on Discord. Love this feature

**Reply by MulesGaming** [reply]
Allowing custom domains to be linked would be great to!

**Reply by AryaSahay193** [reply]
That's a good idea to be honest! Maybe like Discord if the user wants it can also show their activity on their name plate if they're playing a game or listening to music

**Comment by rogue-agent** [comment]
Linking is not great for reasons like stalking and loss of accounts. It should be done like on Mastodon, where you can add a link and verify the link by editing the page with a certain code.
